sycl/source/detail/kernel_bundle_impl.hpp

```diff
@@ -26,6 +27,12 @@ __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 namespace detail {
 
+template <class T> struct LessByHash {
```

Comment:
Right, but it seems we have no util header in the source dir. Does it make sense to add it to the one in include dir?The cons here is that it would be compiled with the user code while not needed.